Timothy Fix (#11094)
endless-sky/endless-sky: data/human/human missions.txt

		near "Zeta Aquilae" 3 10
		government "Republic" "Free Worlds" "Independent" "Pirate" "Neutral"
	destination "Rand"
	to offer

> 		has "remembrance day"

Reviewer: ```suggestion
		has "event: remembrance day"
```

> 			`	As your ship approaches the mine you can't help but notice the burnt out remains of a gutted and abandoned town. A few minutes later your ship rattles down outside the mine's entrance.`

Reviewer 2: Sad to learn [I can't comment above this](https://github.com/github/roadmap/issues/347), but on `line 8002` the opening line says:
```
`Instead of Regina you are met by Timothy's executive secretary, Wynnafrith. She is clearly a little uncomfortable in Rand's regular gravity, but is doing her best to push through it.`
```

It's not overly descriptive and the player might very well have forgotten that Rand has light gravity, so as the first line the player sees for that mission it could be read that Rand has a `normal` gravity that's making Wynnafrith uncomfortable. I'd suggest a change to something making the low gravity aspect more clear, which can be as simple as `regular` --> `low`:

```
`Instead of Regina you are met by Timothy's executive secretary, Wynnafrith. She is clearly a little uncomfortable in Rand's low gravity, but is doing her best to push through it.`
```

Author: Done.

			choice
				`	(Leave your ship.)`

> 			`	It's dark and dusty outside. The planet's gravity makes you feel languid, and the air uncomfortably light in your lungs. The entrance to the mine looms large before you and seems suddenly quite menacing. Perhaps this wasn't the best idea...`

Reviewer 2: ```suggestion
			`	It's dark and dusty outside. The planet's low gravity makes you feel languid, and the air uncomfortably light in your lungs. The entrance to the mine looms large before you and seems suddenly quite menacing. Perhaps this wasn't the best idea...`
```

			choice
				`	(Walk inside.)`

> 			`	The woman before you is middle-aged and unusually gaunt, she has the hollowed out appearance of a person whose lived too long in Rand's frail gravity. She's attractive though, and there is something very familiar about her face. You've met her before...`

Reviewer: ```suggestion
			`	The woman before you is middle-aged and unusually gaunt, with the hollowed out appearance of a person who's lived too long in Rand's frail gravity. She's attractive though, and there is something very familiar about her face. You've met her before...`
```

> 			`	"Captain <last>!" She says with a genuine grin. "I doubt you recognize me, but I was Governor Radrickson's secretary, Wynnafrith. We met years back at your banquet."`

Reviewer: Not changed in this PR, but it could be.
```suggestion
			`	"Captain <last>!" she says with a genuine grin. "I doubt you recognize me, but I was Governor Radrickson's secretary, Wynnafrith. We met years back at your banquet."`
```

> 			`	She glances down at herself as if recognizing her body's transformation from offworld solid to Rander waif. "I dare say the world has changed me now that I'm away from the gravity generators.`

Reviewer: offworld... solid?
Perhaps that could be clarified a bit; it's somewhat confusing.

Author: I agree it isn't ideal, however, I think contextually it makes sense. Someone with a regular/1g amount of muscle mass is colloquially termed an offworld solid. I think other solutions are liable to just draw out the moment in an attempt to be more succinct.

Reviewer: Ah, I didn't see the edit here.
Yes, "healthy offworlder" would probably work better.
```suggestion
			`	She glances down at herself as if recognizing her body's transformation from healthy offworlder to Rander waif. "I dare say the world has changed me now that I'm away from the gravity generators.`
```
